docs/privacy_policy.md

```diff
@@ -0,0 +1,31 @@
+# Privacy Policy
+
+We respect and value your privacy. This Privacy Policy outlines how we collect, use, and protect any personal or anonymous data we may collect from users of our software. By using our software, you agree to the terms of this Privacy Policy.
+
+## Collection of Anonymous User Data
+
+We may collect anonymous user data when you use our software. This information includes a "user ping" of data [posted](https://requests.readthedocs.io/en/latest/user/quickstart/#post-a-multipart-encoded-file) to [pipedream](https://pipedream.com/) every time the software is used. This data helps us understand how we can make our software better users. We may also use this information to demonstrate to agencies and corporations that people are using our software, which may help us to grow the project.
+
+The data we currently collect includes:
+- your IP Address
+- The time the "user ping" was sent 
+- information about the hardware of the cameras
```

Comment:
not true - and also too specific

Just say some thing general and link to the `pipedream pings` class in the repo